arlo_server/rounds.py

```diff
@@ -37,6 +38,7 @@ def serialize_round(r: Round) -> dict:
         "roundNum": r.round_num,
         "startedAt": isoformat(r.created_at),
         "endedAt": isoformat(r.ended_at),
+        "isAuditComplete": is_audit_complete(r.id),
```

Comment:
flagging for later: this means that we're doing a query for each round that is serialized in a list of rounds, not ideal if there are a lot of rounds, probably not an issue right now.

Reply:
Yeah, I thought about that. For most list endpoints that have a computed field like this, I batched into one query, but I figured in this case it's so few rounds that it's a negligible difference.